bin/scotty.js

```diff
@@ -159,10 +164,19 @@ function cmd(console) {
     .then(() => beamUp(args, args.region, console))
 }
 
+
 function beamUp (args, region, console) {
-  return scotty(args.source, args.bucket, region, args.website, args.spa, args.update, args.force, args.quiet, console)
-    .then(endpoint => clipboardy.write(endpoint))
-    .then(() => process.exit(0))
+  const options = Object.assign({
+    logger: console
+  }, args, {region})
+  return scotty(options)
+    .then(endpoint => {
+      return new Promise(resolve => {
+        clipboardy.write(endpoint)
+          .then(() => resolve(true))
+          .catch(err => console.log(`${colors.yellow('Copy to clipboard issue')} ${err.message}`))
```

Comment:
Not sure if you add this but anyway -- I don't think we need to log this issue. Worst case -- it will not be in the clipboard, and that's not real issue.